.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: release
+
+on:
+  # Runs automatically when a tag beginning with 'release-' is pushed.
+  push:
+    tags:
+      - v*
```

Comment:
I think:
```suggestion
      - release-*
```
right?

Reply:
Solved this workaround buy switching away from goreleaser.